packages/settings-library/src/settings_library/email.py

```diff
@@ -0,0 +1,18 @@
+from pydantic.fields import Field
+from pydantic.types import SecretStr
+
+from .base import BaseCustomSettings
+from .basic_types import PortInt
+
+
+class SmtpSettings(BaseCustomSettings):
```

Comment:
Here I am going to be a pain.
since SMTP is an abreviation of Simple Mail Transfer Protocol. why is the class not named ```SMTPSettings``` ?
anyway it's ok... it's just for being picky...